src/App.js

```diff
@@ -24,6 +32,7 @@ const App = () => (
       <span>Users: </span>
       {users.length}
     </p>
+    <PostList content={content} />
```

Comment:
контен плохое название пропсы, непонятно что туда будет передаваться
```suggestion
    <PostList posts={content} />
```